src/tools/gophish_complete.py

```diff
@@ -6,23 +6,22 @@
 flag is provided with a CAMPAIGN_ID that campaign will be completed.
 
 Usage:
-  gophish-complete (--summary | --complete | --auto=CAMPAIGN_ID) [--log-level=LEVEL] SERVER API_KEY
+  gophish-complete (--auto=CAMPAIGN_ID | --complete | --summary ) [--log-level=LEVEL] SERVER API_KEY
   gophish-complete (-h | --help)
   gophish-complete --version
 
-
 Options:
-  SERVER 	--> Full URL to GoPhish server
-  API_KEY 	--> API Access Key
-  -c --complete   Completes Campaign and shows summary
-  -s --summary    Displays Campaign Summary
-  -h --help      Show this screen.
-  --version      Show version.
+  API_KEY                   API Access Key.
+  SERVER                    Full URL to GoPhish server.
+  -a --auto=CAMPAIGN_ID     Complete a specific campaign identified by CAMPAIGN_ID.
+  -c --complete             Complete Campaign and show summary.
+  -s --summary              Displays Campaign Summary.
```

Comment:
Another example of unnecessary capitalization- only the first word should be capitalized.